src/libcore/result.rs

```diff
@@ -309,6 +309,30 @@ impl<T, E> Result<T, E> {
         !self.is_ok()
     }
 
+    /// Returns `true` if the result is an [`Ok`] value containing the given value.
+    ///
+    /// # Examples
+    ///
+    /// ```
```

Comment:
```suggestion
    /// ```
    /// #![feature(option_contains)]
    ///
```